voctogui/ui/voctogui.ui

```diff
@@ -255,7 +255,7 @@
             </child>
             <child>
               <object class="AudioLevelDisplay" id="audiolevel_main">
-                <property name="width_request">30</property>
+                <property name="width_request">100</property>
```

Comment:
Is this with dependent on the number of meters? Should we set it dynamicly from the Code?